content/en/tests/flaky_test_management/auto_test_retries.md

```diff
@@ -73,6 +73,17 @@ This behavior can be fine-tuned with the following environment variables:
 
 In addition to automatically retrying failed tests, Failed Test Replay allows you to see local variable data in the topmost frame of the test error's stack trace. Enable this feature with the **Failed Test Replay** toggle.
 
+#### Create a logs index
+
+Failed Test Replay creates logs that are sent to Datadog and appear alongside your regular application logs.
+
+If you use [Exclusion filters][4], ensure Failed Test Replay logs are not filtered:
+
+1. Create a logs index and [configure it][5] to the desired retention with **no sampling**.
+2. Set the filter to match on the `source:dd_debugger` tag. All Failed Test Replay logs have this source.
+3. Ensure that the new index takes precedence over any other with filters that match that tag, because the first match wins.
```

Comment:
```suggestion
3. Ensure that the new index takes precedence over any other index with filters that match that tag, because the first match wins.
```